content/control-api/curl-examples.textile

```diff
@@ -0,0 +1,205 @@
+---
+title: Examples
+meta_description: "The Control API is a REST API that enables you to manage your Ably account programmatically."
+meta_keywords: "Ably, Control API, account management, control, REST API, OpenAPI Specification, OAS3."
+section: control-api
+index: 20
+jump_to:
+  Help with:
+    - Examples - Apps#examples-apps
+    - Examples - Queues#examples-queues
+    - See also#see-also
+---
+
+This topic includes some examples of using the Control API with "Curl":https://curl.se/.
+
+In the code examples, you will need to set the following variables by any convenient method (such as setting the variables in a script, or copying and pasting suitable values directly into the code):
+
+|_. Variable |_. Description |
+| ACCOUNT_ID | Your Ably Account ID (see "here":/control-api#account-id) |
+| ACCESS_TOKEN | Your Ably access token for the Control API (see "here":/control-api#authentication) |
+| APP_ID | The application ID for the application you want to modify or retrieve information for (see "here":/control-api#app-id) |
+
+h2(#examples-apps). Applications
+
+You can use the Control API to carry out tasks such as listing and creating applications. Operations available include:
+
+* Get a list of apps
+* Create an app
+* Update an app
+* Delete an app
+* Update an app's APNS info 
+
+h3(#examples-apps-list). List your apps
+
+To list all the apps associated with an account:
+
+```[sh]
+curl "https://control.ably.net/v1/accounts/${ACCOUNT_ID}/apps" \
+     -H "Authorization: Bearer ${ACCESS_TOKEN}" \
+     -H "Accept: application/json"
+```
+
+Sample response:
+
+```[json]
+[
+    {
+        "accountId": "VgQpOZ",
+        "id": "28GY6a",
+        "name": "Default",
+        "status": "enabled",
+        "tlsOnly": true,
+        "apnsUseSandboxEndpoint": false,
+        "created": 1602844091113,
+        "modified": 1602844093051
+    },
+    ...
+]
+```
+
+h3(#examples-apps-create). Create an app
+
+To create an app:
+
+```[sh]
+curl --location --request POST 'https://control.ably.net/v1/accounts/${ACCOUNT_ID}/apps' \
+--header 'Content-Type: application/json' \
+--header 'Authorization: Bearer ${ACCESS_TOKEN}' \
+--data-raw '{
+    "name": "Created App 222",
+    "status": "enabled",
+    "tlsOnly": true
+}'
+```
+
+See the "API reference":/control-api/api-reference/#tag/apps/paths/~1accounts~1{account_id}~1apps/post for information on the request body.
+
+Sample response:
+
+```[json]
+{
+    "accountId": "VgQpOZ",
+    "id": "bh4QSw",
+    "name": "Created App 222",
+    "status": "enabled",
+    "tlsOnly": true,
+    "apnsUseSandboxEndpoint": false,
+    "created": 1625813276973,
+    "modified": 1625813276973
+}
+```
+
+h2(#examples-queues). Queues
+
+You can use the Control API to manage Ably queues. The main operations are:
```

Comment:
Can we link to the docs or glossary entry that explains what queues are?

Reply:
We do in the intro, but no harm adding it here as well.